lms/templates/ux/reference/index.html

```diff
@@ -12,10 +16,10 @@
     <div class="inner-wrapper">
         <div class="main-column">
             <article class="window unit-body">
-                <h1>UX Style Reference</h1>
+                <h2>UX Style Reference</h2>
 
                 <section class="xblock xblock-student_view xmodule_display xmodule_HtmlModule">
-                    <h2>Page Types</h2>
+                    <h3>Page Types</h3>
                     <ul>
                         <li><a href="pattern-library-test.html">Pattern Library test page</a></li>
                     </ul>
```

Comment:
These files are only tests for developers, so I wouldn't want to force the translators to have to deal with the strings.